.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "browser": true,
+        "es2020": true,
+        "mocha": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "script",
+        "ecmaFeatures": {
+            "impliedStrict": true
+        }
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "@typescript-eslint/no-explicit-any": "error",
+        "@typescript-eslint/no-var-requires": "off" /** allows require statements outside of imports **/ 
+        /**
+        , "@typescript-eslint/explicit-function-return-type": "error" // required function return types
+        **/
```

Comment:
I left it there because I think it will be a useful rule in a next step.

Reply:
it means that we should discuss adding or not this rule in this PR

Reply:
Ok, so I am "pro" this rule because it makes the return type explicit which will enable type checks for the returned value in case it could be ambigious (like value OR undefined).

Reply:
Given the description when not to use it https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md#when-not-to-use-it it sounds right to have the rule activated.

I see no added value to change from warning to error. it is adding code to maintain for little benefit.

Reply:
@bfitzpat what is your opinion?

Reply:
Let's leave it as a warning for now and test this out.